dataactbroker/scripts/update_historical_duns.py

```diff
@@ -86,24 +125,29 @@ def main():
     args = parser.parse_args()
 
     sess = GlobalDB.db().session
+    client = sams_config_is_valid()
 
     logger.info('Retrieving historical DUNS file')
     start = datetime.now()
     if CONFIG_BROKER["use_aws"]:
         s3connection = boto.s3.connect_to_region(CONFIG_BROKER['aws_region'])
         s3bucket = s3connection.lookup(CONFIG_BROKER["archive_bucket"])
-        duns_file = s3bucket.get_key("DUNS_export.csv").generate_url(expires_in=10000)
+        duns_file = s3bucket.get_key("DUNS_export_deduped.csv").generate_url(expires_in=10000)
```

Comment:
This is not the same spelling as the one you create, it will never find it.

Reply:
Renaming the generated one `DUNS_export_deduped`.